custom_components/econnect_metronet/switch.py

```diff
@@ -0,0 +1,95 @@
+"""Module for e-connect switch (outputs)."""
```

Comment:
You can remove this comment as there is no need to define a module docstring. We may think to propagate this change to other functions as well, but not in this PR.